sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2621,9 +2632,15 @@ def post_teardown_cleanup(self,
         backend_utils.SSHConfigHelper.remove_cluster(handle.cluster_name,
                                                      handle.head_ip,
                                                      auth_config)
+
+        cluster_cost = global_user_state.get_cost_for_cluster(
+            handle.cluster_name)
         global_user_state.remove_cluster(handle.cluster_name,
                                          terminate=terminate)
 
+        if cluster_cost:
+            logger.info(f'Estimated cost of cluster {handle.cluster_name}: '
+                        f'${cluster_cost:.2f}')
```

Comment:
The `logger.info` seems to conflict with our progress bar for termination. Can we move the cost fetching to the `cli.py`, i.e. print the cost after the down or before the down starts?
![image](https://user-images.githubusercontent.com/6753189/204911791-13c901cf-3bfe-4db8-b5af-d5b9c29d8e6c.png)